.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!--
+Thank you for contributing to the Elastic Docs! 🎉
+Use this template to help us efficiently review your contribution.
+-->
+## Summary
+
+Describe what your PR changes or improves.  
+Describe the reason you are making this change. 
+If your PR fixes an issue, link it here.
+
+## Generative AI disclosure
+
+To help us ensure compliance with the Elastic open source and documentation guidelines, please answer the following:
+
+1. Did you use a generative AI (GenAI) tool to assist in creating this contribution?
+- [ ] Yes  
+- [ ] No  
+
+2. If you answered "Yes" to the previous question, please specify the tool(s) and model(s) used (e.g., Google Gemini, OpenAI ChatGPT-4, etc.).
+
+Tool(s) and model(s) used:
+
+---
+
+## Checklist
+
+Before requesting a review, please confirm the following:
+
+- [ ] PR follows the [Elastic Docs contribution guidelines](https://www.elastic.co/docs/contribute-docs)  
+- [ ] Content has been proofread for grammar, spelling, and clarity  
+- [ ] Any code examples have been tested and verified  
+- [ ] Images, screenshots, or diagrams included where necessary  
+- [ ] Links and references are correct and functioning  
+
+--
+
+## Reviewers and timeline
```

Comment:
The table feels overly complex. I also don't really understand the review timelines still. More useful IMO is the PR author saying things like:

@person for technical accuracy in the X section and @team-docs for content and structure
_or_
@team please focus extra attention on the new `### Limitations` section and the updated screenshots.
_or open questions like:_
Are there edge cases or user journeys I'm missing? Anything that might confuse readers?

Essentially being more explicit.

Reply:
I'd vote for de-scoping this PR to do the minimal AI legal compliance thing, and not try to cram general PR hygiene stuff into it TBH